### PR TITLE
Adds a new brain line.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

see title
## Why It's Good For The Game
its a good jokes

### CODE REVIEW

**Comment by Fikou** [comment]
you did indentation with tabs instead of spaces like the other things in that json

File: strings/traumas.json [hunk 1]
```diff
@@ -184,7 +185,8 @@
 	"YOU. STOP.",
 	"REST, MORTALS, TOMORROW IS A LONG DAY.",
 	"YOU MORTALS MAKE ME SICK.",
-	"HONK..."
+	"HONK...",
+	"THERE IS NO SUCH THING AS A COINCIDENCE."
```
Comment (Time-Green) [inline strings/traumas.json:189]: These lines are used by Godwokem Syndrome and contain keywords (such as stop, sick, rest, honk) that cause the voice of god effect. This line doesn’t contain any voice of god keywords and honestly doesn’t fit with the rest of the lines

Consider checking the voice of god to check out what the keywords are and use one of them in this

Comment (Iamgoofball) [reply strings/traumas.json:189]: Whoops, got it, figured they were easter egg lines like the other stuff in the file.

**Comment by coiax** [comment]
I don't get it.

**Pull request closed without merging**